pytorch_lightning/utilities/__init__.py

```diff
@@ -21,13 +22,26 @@
 from pytorch_lightning.utilities.distributed import rank_zero_only, rank_zero_warn, rank_zero_info
 from pytorch_lightning.utilities.parsing import AttributeDict, flatten_dict, is_picklable
 
-try:
-    from apex import amp
-except ImportError:
-    APEX_AVAILABLE = False
-else:
-    APEX_AVAILABLE = True
 
+def _module_available(module_path: str) -> bool:
```

Comment:
nice, this is great. It's what I wanted to do for #2266 but never got around to it.
would be great to follow up on this and use this utility in other places too <3

Reply:
yes, that is why I moved it in the end to PL util even it was developed and used for now in examples :]

Reply:
Yes, let's use `_module_available` everywhere, it looks cleaner than trying to do imports.